lib/scsi_defs.h

```diff
@@ -448,16 +448,16 @@ struct Read32Command {
   uint8_t group_number : 5;
   uint8_t additional_cdb_length : 8;  // 0x18
   uint16_t service_action : 16;       // 0x0009
-  uint8_t rd_protect : 3;
+  uint8_t rdprotect : 3;
   bool dpo : 1;     // disable page output bit
   bool fua : 1;     // Forced Unit access bit
   bool rarc : 1;    // Rebuild Assist Recovery bit
   bool fua_nv : 1;  // Forced Access Unit Non-Volatile
   uint16_t reserved_3 : 9;
   uint64_t logical_block_address : 64;
-  uint32_t expected_initial_logical_block_reference_tag : 32;
-  uint16_t expected_logical_block_application_tag : 16;
-  uint16_t logical_block_application_tag_mask : 16;
+  uint32_t eilbrt : 32;  // expected_initial_logical_block_reference_tag
```

Comment:
renaming for convenience. using the acronym is also consistent with how the docs reference this field